Pipfile.lock

```diff
@@ -128,24 +152,34 @@
             "index": "pypi",
             "version": "==2.23.0"
         },
+        "snap-api-prototype": {
+            "hashes": [
+                "sha256:a1b3d07e2c73a2f04173b48c7e5b6e80b1b61554ecc530db913358d30410efc3",
+                "sha256:a8feb45801c68db2f98528f87b1b652bfd4f035ce18f620aae98a31bb64078f8"
+            ],
+            "index": "test-pypi",
+            "version": "==0.0.1"
+        },
         "urllib3": {
             "hashes": [
-                "sha256:2f3db8b19923a873b3e5256dc9c2dedfa883e33d87c690d9c7913e1f40673cdc",
-                "sha256:87716c2d2a7121198ebcb7ce7cccf6ce5e9ba539041cfbaeecfb641dc0bf6acc"
+                "sha256:3018294ebefce6572a474f0604c2021e33b3fd8006ecd11d62107a5d2a963527",
+                "sha256:88206b0eb87e6d677d424843ac5209e3fb9d0190d0ee169599165ec25e9d9115"
             ],
-            "version": "==1.25.8"
+            "version": "==1.25.9"
         },
         "werkzeug": {
             "hashes": [
-                "sha256:169ba8a33788476292d04186ab33b01d6add475033dfc07215e6d219cc077096",
-                "sha256:6dc65cf9091cf750012f56f2cad759fa9e879f511b5ff8685e456b4e3bf90d16"
+                "sha256:2de2a5db0baeae7b2d2664949077c2ac63fbd16d98da0ff71837f7d1dea3fd43",
+                "sha256:6c80b1e5ad3665290ea39320b91e1be1e0d5f60652b964a3070216de83d2e47c"
             ],
-            "version": "==1.0.0"
+            "version": "==1.0.1"
         }
     },
     "develop": {
         "bandit": {
             "hashes": [
+                "sha256:1d0a4df27713462410157c705c3fea61c59e6e03473e65116e9b1d9659dd426e",
```

Comment:
These additional hashes match the hashes for `bandit` on test.pypi.org: https://test.pypi.org/project/bandit/.

Pipenv seems to add the hashes for dependencies on both sources when more than one source is listed, even though we ultimately only use one source per dependency as specified in the Pipfile.